lib/http-bridge/src/Cardano/Wallet/HttpBridge/Transaction.hs

```diff
@@ -52,10 +54,12 @@ newTransactionLayer = TransactionLayer
     { mkStdTx = \keyFrom inps outs -> do
         let ins = (fmap fst inps)
         let tx = Tx ins outs
+        unless (not (any (\ (TxOut _ c) -> c == Coin 0) outs))
```

Comment:
`when (any ...)` is a bit less-confusing than the double-negation "unless (not (any" I think :smile:

Reply:
yes, I had `when (not. null ...` but hlint **suggested** current version